.ci/targets/android_platform_tests_api_33.yaml

```diff
@@ -0,0 +1,29 @@
+# Same as android_platform_tests.yaml with only packages currently requiring
+# Android 33 due to test failures caused by running on Android 34 AVDs.
+tasks:
+  - name: prepare tool
+    script: .ci/scripts/prepare_tool.sh
+    infra_step: true # Note infra steps failing prevents "always" from running.
+  - name: download Dart and Android deps
+    script: script/tool_runner.sh
+    infra_step: true
+    args: ["fetch-deps", "--android", "--supporting-target-platforms-only"]
+  - name: build examples
+    script: script/tool_runner.sh
+    args: ["build-examples", "--apk"]
+  - name: lint
+    script: script/tool_runner.sh
+    args: ["lint-android"]
+  # Native unit and native integration are split into two steps to allow for
+  # different exclusions.
+  # TODO(stuartmorgan): Eliminate the native unit test exclusion, and combine
+  # these steps.
+  - name: native unit tests
+    script: script/tool_runner.sh
+    args: ["native-test", "--android", "--no-integration", "--exclude=script/configs/exclude_native_unit_android.yaml"]
+  - name: native integration tests
+    script: script/tool_runner.sh
+    args: ["native-test", "--android", "--no-unit", "--filter-packages-to=script/configs/still_requires_api_33_avd_integration.yaml"]
+  - name: drive examples
+    script: script/tool_runner.sh
+    args: ["drive-examples", "--android", "--filter-packages-to=script/configs/still_requires_api_33_avd_drive_examples.yaml"]
```

Comment:
I would suggest having just one config file covering everything that needs an exclusion for any reason, and then passing that config file to *every* task. That way there's a clear and consistent separation between 33 and 34. Without that, it'll be tricky to avoid double-running some tests.